script/Ciel/Day 13/13_01_02_CIEL13_7B.txt

```diff
@@ -538,32 +575,37 @@ He said it was a light that anyone would be proud of.
 [sha:dddd802a1188b51a862a238529f439c9d616ea9d]{
 -- Page 261, Offset 36033.
 -- どれほどの罪を背負うとも、善良な心を持ち、育み、これを捨てる事をしなかったのなら。
-He said that no matter how heavy the sins I bear may be, if I've held onto, fostered, and sustained a virtuous heart―――
+That no matter how heavy the sins I bear may be,
+ if I maintain that virtuous heart――if I nurture it, and never cast it aside―――
 }
 [sha:4083dd00eee4e21c99f572ee9df24479d54a961f]{
 -- Page 261, Offset 36034.
 -- あなたの贖罪は、決して無意味ではなかったと。
-then my atonement was not meaningless by any means.
+Then my atonement will not have been meaningless.
 }
 [sha:a666754143afeea1a57d0a4c89478bfc40d95a67]{
 -- Page 262, Offset 36035.
 -- ……どうしようもないほどの悲しみを隠して。
-...Hiding away the sadness that he couldn't help but feel.
+// This page shuffled around a bit since this sentence is kinda awk on its own
+He spoke while doing his best to hide his crushing sadness...
 }
 [sha:2645263dd34937778d1df3bf0ebbcaffffe519bd]{
 -- Page 262, Offset 36036.
 -- 彼は、自分がほしくてほしくてたまらなかったものを、諦めるようにそう言った。
-He speaks to me as though he's given up on something he so desperately wanted.
+He sounded as if he'd given up on something that he had desperately, desperately wished for.
```

Comment:
Need to keep 'trying to' on the 隠して, otherwise how could she know it was there? Or this just isn't actually his 悲しみ, and she's actually the sad one here

Reply:
No, it's his sadness. And I dunno how she knows, but she clearly does?

>He said all of that while obscuring his own unbearable sadness.